src/main.py

```diff
@@ -280,15 +282,45 @@ def send_email(to_addr, subject, content):
     }
     return ses.send_email(**kwargs)
 
+def email_user__old_credentials_disabled(user_csvrow):
+    ensure('results' in user_csvrow, "`email_user__old_credentials_disabled` requires the results of calling `execute_user_report`")
+    ensure('disable' in user_csvrow['results'] and user_csvrow['results']['disable'],
+           "`email_user__old_credentials_disabled` requires a key was successfully disabled")
+    disabled_credential_key = dict(user_csvrow['actions'])['disable']
+    name, to_addr = vals(user_csvrow, 'name', 'email')
+    subject = 'AWS credentials disabled'
+    content = '''Hello {insert-name-of-human},
+
+Your old credentials "{insert-disabled-credential-key}" have exited their grace period
+
+{insert-grace-period} or more days ago you were issued new AWS credentials to replace
+your old set "{insert-disabled-credential-key}". They have now been disabled.
```

Comment:
could state the subject of the original email, so that it's clear what to look for in an inbox.

Reply:
it's a wip, I hadn't finished writing the message template. There are actually two attempts at the message here.